app/views/collection_drafts/forms/_descriptive_keywords.html.erb

```diff
@@ -145,6 +145,14 @@
   </div>
 </fieldset>
 
+<!-- Passing regex for validation of Value and Data Type fields -->
+<%= hidden_field_tag 'float-regex-for-validation', CollectionDraftsController::FLOAT_REGEX %>
```

Comment:
Not necessarily asking for a change, but just asking: Did you try a script tag? That is the way we pass most data to javascript (see keywords and _I think_ others)

Reply:
I used this [article](http://railscasts.com/episodes/324-passing-data-to-javascript) and considered the script tag option, but thought it may be more complex as I'm not sure if I should define the regex's as variables, returned from a function, or global variable. Nonetheless, if script tags are the norm for us, I think its worth the change to be consistent. Looking for examples in the repo now

Reply:
Yeah I definitely think script tags are the way to go. We use them in a ton of places.